package-build.el

```diff
@@ -463,11 +455,30 @@ Return a cons cell whose `car' is the root and whose `cdr' is the repository."
                             "-d" target-dir repo))))
         (apply 'package-build--run-process dir "cvs" "log"
                (package-build--expand-source-file-list dir config))
-        (or (package-build--find-parse-time-latest "date: \\([0-9]\\{4\\}-[0-9]\\{2\\}-[0-9]\\{2\\} [0-9]\\{2\\}:[0-9]\\{2\\}:[0-9]\\{2\\}\\( [+-][0-9]\\{4\\}\\)?\\)" bound)
-            (package-build--find-parse-time-latest "date: \\([0-9]\\{4\\}/[0-9]\\{2\\}/[0-9]\\{2\\} [0-9]\\{2\\}:[0-9]\\{2\\}:[0-9]\\{2\\}\\);" bound)
-            (error "No valid timestamps found!"))
-        ))))
 
+        ;; `cvs log` does not provide a way to view the previous N
```

Comment:
I realized the `package-build--find-parse-time-latest` function was still being used by the CVS checkout function, so that's why I added this.